src/www/pages/patterns/header-v2/index.njk

```diff
@@ -0,0 +1,560 @@
+          subnavItems: [{
+            name: "Ticket finder"
+          },
+          {
+            name: "Types of ticket"
+          }]
+        },
+        {
+          name: "Discounts and free travel passes",
+          icon: "general-wallet",
+          subnavItems: [{
+            name: "Under 18s"
+          },
+          {
+            name: "Students"
+          },
+          {
+            name: "Families and groups"
+          },
+          {
+            name: "Finding or starting a new job"
+          },
+          {
+            name: "Disabled person's pass"
+          },
+          {
+            name: "Older person's pass"
+          },
+          {
+            name: "WASPI women's pass"
+          }]
+        },
+        {
+          name: "Manage your ticket",
+          icon: "general-ticket",
+          subnavItems: [{
+            name: "Replace your lost ticket or pass"
+          },
+          {
+            name: "Update a Direct Debit"
+          },
+          {
+            name: "Get a refund"
+          },
+          {
+            name: "Make a payment"
+          }]
+        },
+        {
+          name: "Manage your Swift card",
+          icon: "swift-bird-icon",
+          subnavItems: [{
+            name: "Update your personal details"
+          },
+          {
+            name: "Top up your Swift card"
+          },
+          {
+            name: "View your payment history"
+          },
+          {
+            name: "View your travel history"
+          },
+          {
+            name: "How to use your Swift card"
+          },
+          {
+            name: "Replace your lost Swift card"
+          }]
+        }]
+      },{
+        name: "Get help",
+        subnavItems: [{
+          name: "Contact an operator"
+        },
+        {
+          name: "Find a travel centre"
+        },
+        {
+          name: "Find lost property"
+        },
+        {
+          name: "Give feedback"
+        },
+        {
+          name: "Contact us"
+        },
+        {
+          name: "Report anti-social behaviour on public transport"
+        },
+        {
+          name: "Request a new bus shelter or stop pole"
+        },
+        {
+          name: "Report a problem with a shelter or stop"
+        }]
+      }],
+      megaMenu: true,
+      megaMenuDemo: true
+    })
+  ])
+}}
+
+{# With title variant #}
+<h3 id="title-header">With a title variant of header</h3>
+<p>When developing a service you may not need the navigation of the main website. When this is the case, you can place the title of the service in the header (providing their is no other h1 element on the page).</p>
+
+<p>To use the title variant:</p>
+<ul>
+  <li>Add the h1 element and ensure it has the classes <code class="wmnds-website-inline-code">.wmnds-header__title .wmnds-col-1 .wmnds-col-sm-auto</code> on the element</li>
+</ul>
+
+
+{{
+  compExample([
+    wmndsHeaderV2({
+      withTitle: true
+    })
+  ])
+}}
```

Comment:
I don't think this should have the wmnds-header--mega-menu modifier class, as it causes issues to the styling on mobile and will affect live services using it (mainly forms).

Eventually this pattern will die and the new mega menu will replace it, but for now we need to make sure live services can still use the older one.